docs/data-sources/available_ip_address.md

```diff
@@ -0,0 +1,28 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
```

Comment:
@TobiPeterG could you update the readme to explain how you have generated the docs. thanks

Reply:
This is generated with the command "go generate" if I'm not mistaken, it's documented in the main.go

It is also automatically executed in the GitHub ci afaik :)